frontend/src/libs/components/notification-container/notification-container.tsx

```diff
@@ -0,0 +1,22 @@
+import 'react-toastify/dist/ReactToastify.css';
+
+import { ToastContainer } from 'react-toastify';
+
+import {
+  NotificationPosition,
+  NotificationTheme,
+} from '~/libs/enums/components/components.js';
+
+type Properties = {
+  theme?: (typeof NotificationTheme)[keyof typeof NotificationTheme];
+  position?: (typeof NotificationPosition)[keyof typeof NotificationPosition];
```

Comment:
We have `ValueOf` helper type exactly for this purpose
```suggestion
  theme?: ValueOf<typeof NotificationTheme>;
  position?: ValueOf<typeof NotificationPosition>;
```